.freebsd.test.py

```diff
@@ -31,13 +31,18 @@
 
 def main():
     judgeenv.env['runtime'] = {}
-    judgeenv.env['extra_fs'] = {'PERL': ['/dev/dtrace/helper$'], 'RUBY2': ['/dev/dtrace/helper$']}
+    judgeenv.env['extra_fs'] = {
+        'PERL': [{"exact_file": "/dev/dtrace/helper"}],
```

Comment:
Nit: we consistently use `'` for strings except where we need to escape a `'`.

Reply:
Marked unresolved because this is the case for all other strings introduced in this PR. Again, I'm surprised flake8 didn't complain.

Reply:
Fixed. #888 now lints this. Should be all clear.